vyper/parser/global_context.py

```diff
@@ -244,9 +244,7 @@ def _mk_getter_helper(cls, typ, depth=0):
     def mk_getter(cls, varname, typ):
         funs = cls._mk_getter_helper(typ)
         return [
-            """@public\n@constant\ndef %s%s(%s) -> %s: return self.%s%s""" % (
-                varname, funname, head.rstrip(', '), base, varname, tail
-            ) for (funname, head, tail, base) in funs
+            f"@public\n@constant\ndef {varname}{funname}({head.rstrip(', ')}) -> {base}: return self.{varname}{tail}" for (funname, head, tail, base) in funs  # noqa: E501
```

Comment:
@jacqueswww @charles-cooper do you think this is easier to read?
```suggestion
            f"""
@public
@constant
def {varname}{funname}({head.rstrip(', ')}) -> {base}:
    return self.{varname}{tail}
""" for (funname, head, tail, base) in funs
```

Reply:
or @davesque

Reply:
@wschwab do this one

Reply:
Tests fail - I'm assuming it has to do with formatting a multi-line string with a `for`.

Reply:
No worries, we'll skip this one